ee/modules/610-service-with-healthchecks/templates/agent/daemonset.yaml

```diff
@@ -0,0 +1,129 @@
+{{- define "agent_resources" }}
+cpu: 1000m
```

Comment:
is it real consumption? look like to much

Reply:
Yes. We have enabled the user to change the number of workers. An agent can be under high load if there are many objects on the node with it for checks